apis/r/.Rbuildignore

```diff
@@ -23,3 +23,6 @@ tiledbsoma/libtiledbsoma.tar.gz.current
 # vscode
 ^\.vscode$
 ^\.devcontainer$
+^travis-tool\.sh$
+^run\.sh$
+^travis_wait_.*\.log$
```

Comment:
That's a leftover smell from my `run.sh` (initially written for Travis.)  Harmless.  Could remove, could remain.